src/scripts.js

```diff
@@ -97,6 +115,145 @@ function getUserById(id, listRepo) {
   return listRepo.getDataFromID(id);
 }
 
+function windowOnClick(event) {
+  if (event.target.classList.contains('new-hydration-button')) {
+    submitNewHydration(userNowId);
+  }
+  if (event.target.classList.contains('new-activity-button')) {
+    submitNewActivity(userNowId);
+  }
+  if (event.target.classList.contains('new-sleep-button')) {
+    submitNewSleep(userNowId);
+  }
+}
+
+function getTodaysDate() {
+  let today = new Date();
+  let dd = String(today.getDate()).padStart(2, '0');
+  let mm = String(today.getMonth() + 1).padStart(2, '0'); //January is 0!
+  let yyyy = today.getFullYear();
+  return today = yyyy + '/' + mm + '/' + dd;
+}
+
+function submitNewHydration(userNowId) {
+  // let todaysDate = getTodaysDate();
```

Comment:
I know this is code that I contributed to. It is working with getTodaysDate in the PR I am about to submit.

Reply:
Ok I was not sure which route we were taking to reload all of the updated information on the page so I did not touch any thing besides adding the conditionals.